var/spack/repos/builtin/packages/fzf/package.py

```diff
@@ -16,36 +14,30 @@ class Fzf(MakefilePackage):
     homepage = "https://github.com/junegunn/fzf"
     url      = "https://github.com/junegunn/fzf/archive/0.17.5.tar.gz"
 
-    version('0.17.5',   sha256='de3b39758e01b19bbc04ee0d5107e14052d3a32ce8f40d4a63d0ed311394f7ee')
-    version('0.17.4',   sha256='a4b009638266b116f422d159cd1e09df64112e6ae3490964db2cd46636981ff0')
-    version('0.17.3',   sha256='e843904417adf926613431e4403fded24fade56269446e92aac6ff1db86af81e')
-    version('0.17.1',   sha256='9c881e55780c0f56b5a30b87df756634d853bfd3938e7e53cb2df6ed63aa84a7')
-    version('0.17.0-2', sha256='a084415231b452b92a6b8aa87a69c0c02ee59bfe95774bf0d4fcc9a6251ece20')
-    version('0.17.0',   sha256='23569faf64cd6831c09aad7030c8b4bace0eb7a979c580b33cc4e4f9ff303e29')
-    version('0.16.11',  sha256='e3067d4ad58d7be51eba9a35c06518cd7145c0cc297882796c7e40285f268a99')
-    version('0.16.10',  sha256='a6b9d8abcba4239d30201cc7911e9c305a5cd750081ce5cd389f8e7425f4dc93')
-    version('0.16.9',   sha256='dd9434576c68313481613a5bd52dbf623eee37a5c87f7bb66ca71ac8add5ff94')
-    version('0.16.8',   sha256='daef99f67cff3dad261dbcf2aef995bb78b360bcc7098d7230cb11674e1ee1d4')
+    version('0.19.0',   sha256='4d7ee0b621287e64ed450d187e5022d906aa378c5390d8c7c1f843417d2f3422')
+    version('0.18.0',   sha256='5406d181785ea17b007544082b972ae004b62fb19cdb41f25e265ea3cc8c2d9d')
 
     depends_on('go@1.11:')
+    import_resources('resources-0.19.0.json')
```

Comment:
Is there a reason you don't include a `when` constraint? I assume `when='@0.19.0` would be correct. To be clear it's not technically wrong to make this unqualified.

Reply:
The when constraints are included in the json definitions, which are then fed into the resource parser.  I fixed the bad example in the GoPackage docs above.